### DISCUSSION TITLE
Zoom and gathertown links

### DISCUSSION THREAD

**Discussion by olzama** [delph-in/docs#49] [Team Posts]
Dear @delph-in/participants !

We are looking forward to seeing you this coming **Monday, July 19th**, at the Summit!

The **schedule**: https://github.com/delph-in/docs/wiki/Virtual2021Schedule

Zoom link for **Sessions 1&2** (on each day): https://stanford.zoom.us/j/98470609846?pwd=bjJYMUZibFgrRVVVTGRhYThUTU1HQT09

Zoom link for **Session 3** (on each day): https://stanford.zoom.us/j/97721141107?pwd=azRQTFVhajBJN0RuSmlpeE1WcGxQdz09

Gather.town link: https://gather.town/app/odGT09ibCQzYAbDj/DELPH-IN%202021 
password for gather.town: slowandsteady
(Gather.town space generously underwritten by Dan and Stanford.)

**Comment by olzama** [comment]
By the way, the business meeting (which is starting now, more or less (14:15 UTC) is open for all @delph-in/participants . Sorry for not making that clear earlier. Please come :).

**Comment by emilymbender** [comment]
Thanks again @olzama for the fun Gather space! I've now created a tiny Smugmug album for this year's Summit (to which I'm happy to add photos if anyone else took any screen shots they wanted to share).

https://erbonzo.smugmug.com/Travel/DELPH-IN-2021/

For now, it's not password protected, so only photos that are meant to be 'on the open web'. I can add a (minimal) password to make it like the others if that becomes appropriate.

**Comment by olzama** [comment]
Here's a short survey that we'd be grateful if @delph-in/participants filled out. If you could not attend the summit, please feel free to also fill out and say whether the setup that we had prevented you from doing so! You will have the opportunity to offer any suggestions as well.

https://forms.gle/gp5Xkcebu5ULoUMJ7

We will share the results with the next year organizers. 

Please let me know if you cannot access the Google Form.
